diff --git a/node_modules/rn-phone-input-field/dist/main/country-picker/styles/picker.style.js b/node_modules/rn-phone-input-field/dist/main/country-picker/styles/picker.style.js
index f92c2c9..0326846 100644
--- a/node_modules/rn-phone-input-field/dist/main/country-picker/styles/picker.style.js
+++ b/node_modules/rn-phone-input-field/dist/main/country-picker/styles/picker.style.js
@@ -43,7 +43,7 @@ var pickerStyles = function (isDark) {
             fontSize: DIMENSIONS.fontSize.medium,
         },
         eachText: {
-            fontSize: DIMENSIONS.fontSize.large,
+            fontSize: DIMENSIONS.fontSize.small,
         },
         searchInput: {
             flexGrow: 1,
diff --git a/node_modules/rn-phone-input-field/dist/main/phone-input/PhoneInput.js b/node_modules/rn-phone-input-field/dist/main/phone-input/PhoneInput.js
index 6edf716..79b702f 100644
--- a/node_modules/rn-phone-input-field/dist/main/phone-input/PhoneInput.js
+++ b/node_modules/rn-phone-input-field/dist/main/phone-input/PhoneInput.js
@@ -1,4 +1,11 @@
 "use strict";
+import {
+  View,
+  Text,
+  TextInput,
+  Image,
+  TouchableOpacity,
+} from "react-native";
 var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     var desc = Object.getOwnPropertyDescriptor(m, k);
@@ -136,7 +143,17 @@ var PhoneInput = (0, react_1.forwardRef)(function (_a, ref) {
     return (<>
         <react_native_1.View style={[themedStyles.container, containerStyle]}>
           <react_native_1.TouchableOpacity activeOpacity={0.7} onPress={openCountryPicker} style={[themedStyles.flexRow, iconContainerStyle]} accessibilityLabel="Select country code" accessibilityRole="button">
-            <react_native_1.Text style={themedStyles.ft28}>{selectedCountry.icon}</react_native_1.Text>
+            {/* <react_native_1.Text style={themedStyles.ft28}>{selectedCountry.icon}</react_native_1.Text> */}
+            <Image
+  source={{
+    uri: `https://flagcdn.com/w40/${selectedCountry.countryCode.toLowerCase()}.png`,
+  }}
+  style={{
+    width: 24,
+    height: 24,
+    borderRadius: 12,
+  }}
+/>
             <react_native_1.Text style={[themedStyles.ft16, codeTextStyle]}>
               +{selectedCountry.callingCode}
             </react_native_1.Text>
